src/jit/decomposelongs.cpp

```diff
@@ -337,8 +336,6 @@ GenTree* DecomposeLongs::DecomposeLclVar(LIR::Use& use)
     }
     else
     {
-        noway_assert(varDsc->lvLRACandidate == false);
```

Comment:
Once we've used a `GT_LCL_FLD` on this, we can't make this a register candidate. Is there somewhere else that this is getting disabled?